### DISCUSSION TITLE
how can i expand the interaction area for the handles?

### DISCUSSION THREAD

**Discussion by RanitManik** [xyflow/xyflow#4828] [Q&A]
I am using custom nodes and edges and I can use the `interactionWidth` prop to change the interaction area for the custom edges:

```javascript
<BaseEdge
    id={id}
    path={edgePath}
    {...props}
    interactionWidth={20}
/>
```
But i could not find a way to do that same for the Handles. How can i do that??

**Comment by bcakmakoglu** [comment]
[You can try to do something like this.](https://codesandbox.io/p/sandbox/gifted-thunder-789n22?file=%2Fsrc%2Fnodes%2FPositionLoggerNode.tsx%3A28%2C24)
